content/source/docs/enterprise/private/vault.html.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: "enterprise2"
+page_title: "Private Terraform Enterprise Customer Managed Vault (Installer Beta)"
+sidebar_current: "docs-enterprise2-private-vault"
+---
+
+# Private Terraform Enterprise Customer Managed Vault
+
+Private Terraform Enterprise can be configured to use an external Vault cluster
+rather than the internal Vault instance. Within PTFE, Vault is used to
+encrypt sensitive information such as variables and states.
+
+An external Vault cluster allows the customer to have full control over how
+the Vault cluster is managed, for example how it is sealed and unsealed, replicated, etc.
+
+When an external Vault cluster is configured along with the External Services installation mode,
+the product becomes fully stateless and can be run in a hot-standby
+configuration to provide failover.
+
+## Setup
+
+Follow these steps to configure an external Vault instance:
```

Comment:
"Use the following as a guide" may be a little softer.  For example, I can see a customer wanting to apply more rigor to the policy name.

Reply:
@KFishner ?